### DISCUSSION TITLE
Subfactions can have different cameos for the same unit/building/power

### DISCUSSION THREAD

**Discussion by Ho6org** [Phobos-developers/Phobos#1038] [Ideas and feedback]
### First check

- [X] The suggestion is **NOT** related to functionality of Ares, HAres, Kratos or any other YR engine extension.
- [X] The suggested feature / enhancement is **NOT** implemented in Ares/Phobos yet and the existing possibilites are not viable or good enough to achieve the wanted result.
- [X] I agree to elaborate the details if requested and provide thorough testing if the feature is implemented.
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search and read the issue list to find a similar issue and didn't find it.
- [X] I thought out how the feature / enhancement should work and described it in the most detailed and clear way.

### Description

For example two soviet subfactions, can have different conscript cameos

**Comment by Daiki9527** [comment]
You can make two units,each for each subfaction,that save a lot works for developers.
